src/components/CalendarDay.jsx

```diff
@@ -48,6 +55,9 @@ export default class CalendarDay extends React.Component {
         onClick={e => this.onDayClick(day, e)}
       >
         <span className="CalendarDay__day">{day.format('D')}</span>
+        {details && (
+          <span className="CalendarDay__details">{details}</span>
```

Comment:
Also, should I add a modifier class to the whole cell when details are present to make styling the content easier?